src/App.js

```diff
@@ -1,10 +1,27 @@
+import { useState } from "react";
 
-import './App.css';
+import "./App.css";
+import "./components/Header";
+import Header from "./components/Header";
+import TaskForm from "./components/TaskForm";
+import GetTasks from "./components/GetTasks";
+import axiosInstance from "./api/axios";
 
 function App() {
+  const [todos, setTodos] = useState([]);
+  const getTask = async () => {
+    try {
+      const data = await axiosInstance.get();
+      setTodos(data.data);
+    } catch {
+      console.log("Error fetching data");
+    }
+  };
```

Comment:
I believe you should call the `getTask` function in a `useEffect` in this file.

Reply:
But your `getTask` function is defined in your `App.js`., why didn't you just call the `getTask` in a `useEffect` there?

Reply:
ok

Reply:
> But your `getTask` function is defined in your `App.js`., why didn't you just call the `getTask` in a `useEffect` there?

I am using useEffect inside the GetTask components already which should handle this. I just feel the useEffect should not flood the App.js since its already in the GetTask

Reply:
If I'm correct, you put `getTask` in your App.js because other components need it.

Why not also call the function, so that other components have the data before they render?

Reply:
ok. I get now. I will change this